app/form_models/jobseekers/account_transfer_form.rb

```diff
@@ -0,0 +1,20 @@
+class Jobseekers::AccountTransferForm < BaseForm
+  attr_accessor :email, :account_merge_confirmation_code
+
+  validates :email, presence: true
+  validates :email, email_address: true, if: -> { email.present? }
```

Comment:
This validation is being removed in @starswan's PR: https://github.com/DFE-Digital/teaching-vacancies/pull/7066
It seems whichever goes in first, the other PR will need to be updated!